comms-commands/orb.yml

```diff
@@ -1,6 +1,36 @@
 version: 2.1
 description: "Orb with commonly used commands by the Comms Platform Service pipelines"
 
+orbs:
+  terraform: ovotech/terraform-v2@2
+
+jobs:
+  terraform_apply:
+    parameters:
+      path:
+        description: "The path to the terraform code (usually ./terraform/)"
+        type: string
+      environment:
+        description: "This can be either one of (prd, uat or lt)"
+        type: enum
+        enum: [ "prd", "uat", "lt" ]
+    executor: terraform/default
+    steps:
+      - checkout
+      - terraform/apply:
+          path: << parameters.path >>
+          workspace: main
+          backend_config_file: ./environments/<< parameters.environment >>/<< parameters.environment >>.backend
```

Comment:
This forces us to use this file structure in every service, but I think that's a good thing.

Reply:
not sure about this, ideally we should be free to split configs or change the structure when necessary

Reply:
Could have optional param then